tests/security/swtpm/swtpm_legacy_verify.pm

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2020-2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Ship the "swtpm" software TPM emulator for QEMU,
+#          test Legacy guest OS under libvirt, cover both
+#          TPM 1.2 and TPM 2.0
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#81256, tc#1768671
+
+use base 'opensusebasetest';
+use swtpmtest;
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+    foreach my $i ("swtpm_1", "swtpm_2") {
+        start_swtpm_vm($i, "legacy");
+        assert_script_run("sleep 90");    # OS can booted up within 60s, so wait for 90s here to make sure it is up
```

Comment:
here is necessary to use sleep, right? or is there any replacement.
As we know that we will avoid sleep() usage in CONTRIBUTING.md .
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11431/files

Reply:
good question here, let us talk about it to find a better way. listing to the ssh port?

Reply:
fixed